### DISCUSSION TITLE
[GSI_LOGGER]: The given client ID is not found.

### DISCUSSION THREAD

**Discussion by nazariidmytrak** [refinedev/refine#3980] [Q&A] [answered]
Got this error when trying to log in with Google in my application build with Refine, in the examples auth-google-login also does not work and gives the same error. Can you help me?

**Comment by yildirayunlu** [comment]
Hey @nazariidmytrak I tried the [example](https://refine.dev/docs/examples/auth-provider/google-auth/) here but didn't run into any problems. What error are you getting? 

This issue may be related to node version.

**Comment by nazariidmytrak** [comment, marked as answer]
The error was on my side, sorry for disturbing and thank you!

**Reply by agajewskiinnpro** [reply]
What error was it? This information could be helpful to other people.
